tgui/packages/tgui/interfaces/CentcomPodLauncher.js

```diff
@@ -1,518 +1,1125 @@
+import { classes } from 'common/react';
+import { CreateUuid } from 'common/uuid';
 import { multiline } from 'common/string';
-import { Fragment } from 'inferno';
-import { useBackend } from '../backend';
-import { Button, LabeledList, NoticeBox, Section } from '../components';
+import { useBackend, useLocalState } from '../backend';
+import { Flex, ByondUi, Divider, Section, Tabs, Knob, Box, Button, Fragment, ProgressBar, NumberInput, Icon, Input, LabeledList, LabeledControls } from '../components';
 import { Window } from '../layouts';
+import { storage } from 'common/storage';
+import { toFixed } from 'common/math';
 
-export const CentcomPodLauncher = () => {
+const pod_grey = {
+  color: 'grey',
+};
+
+export const CentcomPodLauncher = (props, context) => {
+  const [compact, toggleCompact] = useLocalState(context, 'compact', 0);
   return (
     <Window
-      title="Config/Launch Supply Pod"
-      width={700}
-      height={700}
-      resizable>
-      <Window.Content scrollable>
-        <CentcomPodLauncherContent />
-      </Window.Content>
+      resizable
+      key={compact}
+      title={compact === 0
+        ? "Supply Pod Menu (Use against Helen Weinstein)"
+        :"(Use against Helen Weinstein)"}
+      overflow="hidden"
+      width={compact ? 420 : 690}
+      height={compact ? 360 : 420}>
+      <CentcomPodLauncherContent />
     </Window>
   );
 };
 
-// This is more or less a direct port from old tgui, with some slight
-// text cleanup. But yes, it actually worked like this.
-export const CentcomPodLauncherContent = (props, context) => {
-  const { act, data } = useBackend(context);
+const CentcomPodLauncherContent = (props, context) => {
+  const [compact, toggleCompact] = useLocalState(context, 'compact', 0);
+  const marginer = 0.5;
+  return (
+    <Window.Content>
+      <Flex direction="column" height="100%">
+        <Flex.Item
+          minHeight={compact ? "65px" : "55px"}
+          height={compact ? "42%" : "19%"}
```

Comment:
This would probably be better handled by the `grow` property, but I wouldn't touch it since it works.